Documentation changes for the 2FA additions (#40411)
keycloak/keycloak: docs/documentation/upgrading/topics/changes/changes-26_3_0.adoc

This prevents accidentally downgrading functionality, for example, by using an old `cache-ispn.xml` file from a previous version.

=== Default browser flow changes 2FA to include WebAuthn and Recovery Codes

> Previously the default *browser* flow had a *Browser - Conditional OTP* conditional sub-flow that enabled One-Time Password (OTP) as a 2nd Factor Authentication (2FA). Since this version, the sub-flow has been renamed to *Browser -Conditional 2FA*, the *OTP Form* is _Alternative_, and includes two more 2FA methods: *WebAuthn Authenticator* and *Recovery Authentication Code Form*. Both new executions are _Disabled_ by default, but they can be set to _Alternative_ to include them easily into the flow.

Reviewer: ```suggestion
Previously the default *browser* flow had a *Browser - Conditional OTP* conditional sub-flow that enabled One-Time Password (OTP) as a 2nd Factor Authentication (2FA). Starting with this version, the sub-flow is renamed to *Browser -Conditional 2FA*, the *OTP Form* is _Alternative_, and includes two more 2FA methods: *WebAuthn Authenticator* and *Recovery Authentication Code Form*. Both new executions are _Disabled_ by default, but they can be set to _Alternative_ to include them into the flow.
```
